<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>Graduate Research</title>
    <link rel="shortcut icon" type="image/jpg" href="img/favicon.ico" />
    <link
      href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/css/bootstrap.min.css"
      rel="stylesheet"
      integrity="sha384-gH2yIJqKdNHPEq0n4Mqa/HGKIhSkIHeL5AyhkYV8i59U5AR6csBvApHHNl/vI1Bx"
      crossorigin="anonymous"
    />
    <link
      rel="stylesheet"
      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.9.1/font/bootstrap-icons.css"
    />
    <link rel="stylesheet" href="subpage.css" />
  </head>
  <body>
    <div class="customized_container">
      <div class="title">
        <h1>Graduate Research</h1>
        <a class="rp_outside_link" href="http://pughlab.mbg.cornell.edu/">
          <h5>
            In Pugh Lab, Cornell University
            <i class="bi bi-box-arrow-up-right"></i></h5
        ></a>
      </div>
      
      &nbsp;
      <hr />
      &nbsp;

      <!--research 01-->
      <div class="col">
        <div class="row">
          <div class="col-lg-4">
            <h2>01</h2>
            <h5>
              A streamlined ChIP-exo 6.1 assay measures same-molecule positional
              relationship between transcription factors and nucleosomes in vivo
            </h5>
          </div>
          <div class="col-lg-8">
            <h6>Abstract</h6>
            <p>
              Transcription factors (ssTFs) bind specific DNA sequences in
              promoter and enhancer regions in the context of an organized
              nucleosome architecture. There they recruit cofactors that
              regulated chromatin accessibility and transcription complex
              assembly. ChIP-exo is a high resolution genome-wide assay that
              measures protein-DNA interactions with single-bp precision. Here
              we couple the precision of ChIP-exo with a nuclease chromatin
              fragmentation strategy that cleaves and marks the rotationally
              exposed DNA on nucleosome surfaces. This new technique maps the
              positional orientation of TFs with an adjacent nucleosome surface
              on the same DNA molecule in human cells. Additional modifications
              to the ChIP-exo assay (version 6.1) greatly simplifies library
              construction over prior versions and provides a facile,
              high-precision means to map genomic TF binding sites.
            </p>
          </div>
        </div>
      </div>

      &nbsp;
      <hr />
      &nbsp;

      <!--research 02-->
      <div class="col">
        <div class="row">
          <div class="col-lg-4">
            <h2>02</h2>
            <h5>
              Molecular architecture on cis-regulatory region in human genome
            </h5>
          </div>
          <div class="col-lg-8">
            <h6>Abstract</h6>
            <p>
              Human gene transcription under the control from core promoter and
              cis-regulatory elements proximal or distal from their target
              genes. At those gene regulatory elements, there are motifs can be
              recognized by sequence specific transcription factor, where they
              can interact with other TF or cofactors to increase chromatin
              accessibility or help assembly of the transcription machinery,
              therefore augment the transcription level at transcription
              initiation step. Based on the hypothesis that tissue specific
              expression programs is related to tissue specific molecular
              composition and architecture on cis-regulatory elements, we are
              curious about the details of in-vivo interplay between
              sequence-specific transcription factors (TFs) and other proteins
              that involved in transcription initiation step, we would like to
              use ChIP-exo assays to reveal their positional organization within
              the complexes, therefore understanding how genomic and epigenomic
              information directs gene initiation at cell specific manner.
            </p>
          </div>
        </div>
      </div>

      &nbsp;
      <hr />
      &nbsp;

      <!--research 03-->
      <div class="col">
        <div class="row">
          <div class="col-lg-4">
            <h2>03</h2>
            <h5>Single cell ChIP-exo by split-pool barcoding</h5>
          </div>
          <div class="col-lg-8">
            <h6>Abstract</h6>
            <p>
              Most methods that study the landscape of TF binding including
              ChIP-exo assay measure the ensemble TF signals across millions of
              cells. Though protein-DNA interactions with single-bp precision
              can be achieved currently, it is still obscure critical
              information about the epigenome information of any given cell. In
              this research, we combine nuclei barcoding with ChIP-exo assay to
              profile TF binding at base-bp resolution as well as single cell
              resolution. Here in insoluble nuclei, the fragmented chromatins
              are lambda exonuclease digested and each cell gets a unique
              barcode by nuclei split-pool. After pooling barcoded nuclei for
              immunoprecipitation, the sequence information will demultiplex the
              TF profiling at single-cell resolution.
            </p>
          </div>
        </div>
      </div>
    </div>

    <script
      src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/js/bootstrap.bundle.min.js"
      integrity="sha384-A3rJD856KowSb7dwlZdYEkO39Gagi7vIsF0jrRAoQmDKKtQBHUuLZ9AsSv4jD4Xa"
      crossorigin="anonymous"
    ></script>
    <script>
      w3IncludeHTML();
    </script>
  </body>
</html>
