<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>Graduate Research</title>
    <link rel="shortcut icon" type="image/jpg" href="img/favicon.ico" />
    <link
      href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/css/bootstrap.min.css"
      rel="stylesheet"
      integrity="sha384-gH2yIJqKdNHPEq0n4Mqa/HGKIhSkIHeL5AyhkYV8i59U5AR6csBvApHHNl/vI1Bx"
      crossorigin="anonymous"
    />
    <link
      rel="stylesheet"
      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.9.1/font/bootstrap-icons.css"
    />
    <link rel="stylesheet" href="subpage.css" />
  </head>
  <body>
    <div class="customized_container">
      <div class="title">
        <h1>Graduate Research</h1>
        <a class="rp_outside_link" href="http://pughlab.mbg.cornell.edu/">
          <h5>
            In Pugh Lab, Cornell University
            <i class="bi bi-box-arrow-up-right"></i></h5
        ></a>
      </div>
      
      &nbsp;
      <hr />
      &nbsp;

      <!--research 01-->
      <div class="col">
        <div class="row">
          <div class="col-lg-4">
            <h2>01</h2>
            <h5>
              Translational and rotational setting of nucleosomes across a human genome
            </h5>
          </div>
          <div class="col-lg-8">
            <h6>Abstract</h6>
            <p>
              Understanding how transcription factors (TFs) access DNA within 
              chromatin is central to gene regulation. TFs recognize their 
              DNA binding site either when it is nucleosome-free or its helical 
              site is rotationally accessible on a nucleosome surface. These 
              are manifested as translational and rotational phasing, 
              which are largely undefined in human cells. We developed 
              Benzonase-seq to measure genome-wide phasing and enhanced ChIP-exo (v6) 
              to assess phasing on DNA bound by TFs, including transient ones. 
              We show that phasing differs between bound and unbound sites. 
              For example, unbound CTCF sites are encased within DNA-encoded 
              rotationally phased nucleosomes. But there is no consensus translational 
              phasing until CTCF binds. Similar relationships exist with other 
              TFs like ZKSCAN1, and for transiently binding TFs like FoxA and NFIA. 
              Our approach also refines nucleosome maps near promoters and reveals 
              subnucleosomal structures. These findings offer new insights into how 
              TFs interact with nucleosomes to shape the chromatin landscape. 
            </p>
          </div>
        </div>
      </div>

      &nbsp;
      <hr />
      &nbsp;

      <!--research 02-->
      <div class="col">
        <div class="row">
          <div class="col-lg-4">
            <h2>02</h2>
            <h5>
              An integrated genome-wide mechanism of RNA polymerase II initiation complex assembly
            </h5>
          </div>
          <div class="col-lg-8">
            <h6>Abstract</h6>
            <p>
              How transcription pre-initiation complexes (PICs) assemble on natural 
              promoters remains poorly understood. Here we propose a molecular 
              mechanism governing PIC assembly at human promoters genome-wide 
              that integrates transcription start sites, TATA boxes, +1 nucleosomes, 
              and transcription factor binding. We find that +1 nucleosomes have
              robust DNA-encoded rotational phasing. This, in conjunction with
              TFIID (TAF3) interactions and activator (e.g., SP1, GABPA and NFYC) 
              interactions with TFIID (TAF4) and TFIIA concentrate and orient 
              the transcription machinery within DNA gyres. Such placement positions
              RNA polymerase (Pol) II to conduct a tightly focused search for the 
              optimal initiator dinucleotide. Pol II then initiates transcription 
              and pauses at the +1 nucleosome without disrupting its rotational phase 
              unless the nucleosome is unstable. Together, these findings define key 
              organizational steps that govern PIC assembly, transcription initiation, 
              and pausing within the natural chromatin landscape of promoters.
            </p>
          </div>
        </div>
      </div>

      &nbsp;
      <hr />
      &nbsp;

      <!--research 03-->
      <div class="col">
        <div class="row">
          <div class="col-lg-4">
            <h2>03</h2>
            <h5>Single cell ChIP-exo by split-pool barcoding</h5>
          </div>
          <div class="col-lg-8">
            <h6>Abstract</h6>
            <p>
              Most methods that study the landscape of TF binding including
              ChIP-exo assay measure the ensemble TF signals across millions of
              cells. Though protein-DNA interactions with single-bp precision
              can be achieved currently, it is still obscure critical
              information about the epigenome information of any given cell. In
              this research, we combine nuclei barcoding with ChIP-exo assay to
              profile TF binding at base-bp resolution as well as single cell
              resolution. Here in insoluble nuclei, the fragmented chromatins
              are lambda exonuclease digested and each cell gets a unique
              barcode by nuclei split-pool. After pooling barcoded nuclei for
              immunoprecipitation, the sequence information will demultiplex the
              TF profiling at single-cell resolution.
            </p>
          </div>
        </div>
      </div>
    </div>

    <script
      src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/js/bootstrap.bundle.min.js"
      integrity="sha384-A3rJD856KowSb7dwlZdYEkO39Gagi7vIsF0jrRAoQmDKKtQBHUuLZ9AsSv4jD4Xa"
      crossorigin="anonymous"
    ></script>
    <script>
      w3IncludeHTML();
    </script>
  </body>
</html>
