<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>Graduate Research</title>
    <link rel="shortcut icon" type="image/jpg" href="img/favicon.ico" />
    <link
      href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/css/bootstrap.min.css"
      rel="stylesheet"
      integrity="sha384-gH2yIJqKdNHPEq0n4Mqa/HGKIhSkIHeL5AyhkYV8i59U5AR6csBvApHHNl/vI1Bx"
      crossorigin="anonymous"
    />
    <link
      rel="stylesheet"
      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.9.1/font/bootstrap-icons.css"
    />
    <link rel="stylesheet" href="subpage.css" />
  </head>
  <body>
    <div class="Research_publications_container">
      <div class="title">
        <h1>Research Publications</h1>
      </div>
      &nbsp;
      <div class="col">
        <div class="row rp_content">
          <div class="col-4 text_left">
            <h2>What do transcription factors interact with?</h2>
            &nbsp;
            <h6>Haining Chen and B. Franklin Pugh</h6>
            <p>
              Department of Molecular Biology and Genetics, Cornell University,
              Ithaca, NY 14853, USA Corresponding author
            </p>
            &nbsp;
            <h6>Abstract</h6>
            <p>
              Although we have made significant progress, we still possess a
              limited understanding of how genomic and epigenomic information
              directs gene expression programs through sequence-specific
              transcription factors (TFs). Extensive research has settled on three
              general classes of TF targets in metazoans: promoter accessibility
              via chromatin regulation (e.g., SAGA), assembly of the general
              transcription factors on promoter DNA (e.g., TFIID), and recruitment
              of RNA polymerase (Pol) II (e.g., Mediator) to establish a
              transcription pre-initiation complex (PIC). Here we discuss TFs and
              their targets. We also place this in the context of our current work
              with Saccharomyces (yeast), where we find that promoters typically
              lack an architecture that supports TF function. Moreover, yeast
              promoters that support TF binding also display interactions with
              cofactors like SAGA and Mediator, but not TFIID. It is unknown to
              what extent all genes in metazoans require TFs and their cofactors.
            </p>
            <p>
              <h6>Keywords:</h6> Gene regulation models, Transcription mechanisms,
              Enhancers, Promoters.
            </p>
            &nbsp;
            <p>
              <a
                class="rp_outside_link"
                href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8184585/"
                >Chen, H., & Pugh, B. F. (2021). What do transcription factors
                interact with?. Journal of Molecular Biology, 166883.
                <i class="bi bi-box-arrow-up-right"></i
              ></a>
            </p>
          </div>
          <div class="col-8 text_right">
            <div class="row text_right_content">
              <div class="col">
                <h5>Figure 1</h5>
                <img class="Figure1" src="img/Picture1.png" />
                <p>
                  Fig. 1. Simplified model of enhancer-core promoter interactions.
                  Enhancers contain specific sequence motifs that are bound by TFs
                  . They activate core promoters by regulating promoter/enhancer
                  interactions, PIC assembly, and Pol II recruitment through a
                  wide variety of cofactors.
                </p>
              </div>
              <div class="col">
                <h5>Figure 2</h5>
                <img class="Figure1" src="img/Picture2.png" />
                <p>
                  Fig. 2. Condensate-based model of enhancer-promoter control. TFs
                  with their intrinsically disordered regions (IDRs) along with
                  other IDRs in cofactors coalesce and demix, thereby dramatically
                  increasing the local concentration of components of the
                  transcription machinery. Potential consequences include
                  increased nucleosome dynamics affecting enhancer/promoter
                  accessibility, more frequent PIC assembly, and a steady stream
                  of Pol II loading and firing into an elongation complex.
                </p>
              </div>
            </div>
            &nbsp;
            <div class="row figure3_content">
              <h5>Figure 3</h5>
              <div class="col-lg-8 figure3_img">
                <img class="Figure2" src="img/Picture3.png" />
              </div>
              <div class="col-lg-4 figure3_text">
                Fig. 3. Binary architecture of yeast promoters. Summary schematic
                of yeast chromatin structures with key architectural arrangements
                at constitutive (upper) and induced (lower) promoter classes. T,
                A, B, E, F, H denote TBP, TFIIA, TFIIB, TFIIE, TFIIF, and TFIIH,
                respectively. NFR and NDR denote nucleosome-free region and
                nucleosome-depleted region, respectively.
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>

    <div class="col">
        <hr /> <!-- Optional divider -->

        <div class="row rp_content">
          <div class="col-4 text_left">
            <h2>Genome-wide rotational and translational phasing of nucleosomes with human transcription factors</h2>
            &nbsp;
            <h6>Haining Chen, Jordan E. Krebs, Olivia W. Lang, Devin Melini, Judith Hyle, Chunliang Li, William K. M. Lai, B. Franklin Pugh</h6>
            <p>
              Department of Molecular Biology and Genetics, Cornell University, Ithaca, New York, 14853, USA.
            </p>
            &nbsp;
            <h6>Abstract</h6>
            <p>Understanding how transcription factors (TFs) access DNA within chromatin is central to gene regulation. 
               TFs recognize their DNA binding site either when it is nucleosome-free or its helical site is rotationally 
               accessible on a nucleosome surface. These are manifested as translational and rotational phasing, which 
               are largely undefined in human cells. We developed Benzonase-seq to measure genome-wide phasing and enhanced 
               ChIP-exo (v6) to assess phasing on DNA bound by TFs, including transient ones. We show that phasing differs 
               between bound and unbound sites. For example, unbound CTCF sites are encased within DNA-encoded rotationally 
               phased nucleosomes. But there is no consensus translational phasing until CTCF binds. Similar relationships 
               exist with other TFs like ZKSCAN1, and for transiently binding TFs like FoxA and NFIA. Our approach also 
               refines nucleosome maps near promoters and reveals subnucleosomal structures. 
               These findings offer new insights into how TFs interact with nucleosomes to shape the chromatin landscape.</p>
            <p>
              <h6>Keywords:</h6> Nucleosome phasing, Transcription factors, Chromatin accessibility.
            </p>
            &nbsp;

            <p>
              <a
                class="rp_outside_link"
                href="#"
                >Full citation or link to article
                <i class="bi bi-box-arrow-up-right"></i>
              </a>
            </p>
          </div>

          <div class="col-8 text_right">
            <div class="row text_right_content">
              <div class="col">
                <h5>Figure 1</h5>
                <img class="Figure1" src="img/SecondPaperFig1.png" />
                <p>Fig. 1. Schematic overview of Benzonase-seq (upper) and ChIP-exo v6 (lower) assays represented in the resulting
                  libraries and analyses. Spikes in 10 bp intervals depict cleavages occurring on rotationally phased nucleosomal DNA.</p>
              </div>
              <div class="col">
                <h5>Figure 2</h5>
                <img class="Figure2" src="img/SecondPaperFig2.png" />
                <p>Fig. 2. left: CTCF engages translationally phased nucleosomes. 
                  right: NFIA binding to its DNA site organizes a phased nucleosome downstream.
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>

    <div class="col">
      <hr /> <!-- Optional divider -->

      <div class="row rp_content">
        <div class="col-4 text_left">
          <h2>An integrated genome-wide mechanism of RNA polymerase II initiation complex assembly</h2>
          &nbsp;
          <h6>Haining Chen, Olivia Lang, William K.M. Lai and B. Franklin Pugh</h6>
          <p>
            Department of Molecular Biology and Genetics, Cornell University, Ithaca, New York, 14853, USA.
          </p>
          &nbsp;
          <h6>Abstract</h6>
          <p>How transcription pre-initiation complexes (PICs) assemble on natural promoters remains poorly understood. 
            Here we propose a molecular mechanism governing PIC assembly at human promoters genome-wide that integrates 
            transcription start sites, TATA boxes, +1 nucleosomes, and transcription factor binding. We find that +1 
            nucleosomes have robust DNA-encoded rotational phasing. This, in conjunction with TFIID (TAF3) interactions 
            and activator (e.g., SP1, GABPA and NFYC) interactions with TFIID (TAF4) and TFIIA concentrate and orient 
            the transcription machinery within DNA gyres. Such placement positions RNA polymerase (Pol) II to conduct 
            a tightly focused search for the optimal initiator dinucleotide. Pol II then initiates transcription and 
            pauses at the +1 nucleosome without disrupting its rotational phase unless the nucleosome is unstable. 
            Together, these findings define key organizational steps that govern PIC assembly, transcription initiation, 
            and pausing within the natural chromatin landscape of promoters.</p>
          <p>
            <h6>Keywords:</h6> Pre-initiation complexes, Transcription factors, +1 nucleosome.
          </p>
          &nbsp;

          <p>
            <a
              class="rp_outside_link"
              href="#"
              >Full citation or link to article
              <i class="bi bi-box-arrow-up-right"></i>
            </a>
          </p>
        </div>

        <div class="col-8 text_right">
          <div class="row text_right_content">
            <div class="col">
              <h5>Figure 1</h5>
              <img class="Figure1" src="img/ThirdPaperFig1.png" />
              <p>Fig. 1. A molecular model of PIC architecture and assembly at natural genomic promoter sites.
                Upstream ssTFs bind their cognate TFBS and act uni- or bi-directionally within bidirectional promoter 
                regions, depending on the ssTF. They (e.g., GABPA and SP1) interact with, and locally concentrate, GTFs
                and cofactors within the 3D space of a promoter region without dictating PIC placement. TFIID remains 
                anchored at the dinucleotide-encoded +1 nucleosome via TAF3, where it can reload TBP. This mechanism 
                coordinates upstream ssTFs with a rotationally phased downstream +1 nucleosome so that PIC assembly can 
                be hemispherically aligned to one face of the DNA helix. Core promoter elements and their associated TFs
                (e.g., YY1) might then position the PIC within a defined helical gyre.  </p>
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>

    <script
      src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/js/bootstrap.bundle.min.js"
      integrity="sha384-A3rJD856KowSb7dwlZdYEkO39Gagi7vIsF0jrRAoQmDKKtQBHUuLZ9AsSv4jD4Xa"
      crossorigin="anonymous"
    ></script>
    <script>
      w3IncludeHTML();
    </script>
  </body>
</html>
